README.md

```diff
@@ -130,6 +131,19 @@ associated to `linux` and `apache-servers` subscriptions.
   }
 ```
 
+### Basic Sensu CLI
+
+The following example will manage the resources necessary to use `sensuctl`.
+
+```puppet
+class { '::sensu::cli':
+  url_host => 'sensu-backend.example.com',
+  password => 'supersecret',
+}
+```
+
+**NOTE**: The `sensu::backend` class calls the `sensu::cli` class so it is only necessary to directly call the `sensu::cli` class on hosts not using the `sensu::backend` class.
+
```

Comment:
We should further explain what bootstrapping entrails and why it is needed.

Reply:
The bootstrapping only happens on fresh install of backend. It's something we abstract away and not something I would expect anyone to need to worry about since we just recommend not using default password. The bootstrapping is not needed for just using cli class like this section documents. This is kinda of like a private type functionality. I can document but if module behaves correctly people should be unaware of bootstrap step.

Reply:
Also I hope to rework some the logic to avoid need to pass bootstrap parameters. The current process is semi fragile per #1163. Somewhere still have to hardcode the default bootstrap password though.

Reply:
This can be a commented out section in the code or the README as you see fit, though a few sentences regarding How Sensu ships with a default password and we have to go through this bootstrap process to change the password would help users understand what is happening.

Reply:
I improved the bootstrap process so it's entirely hidden. We document using non-default password and everything else is handled by the provider for `sensu_configure`.  See d2ef64b07805262733c87ccd2bd28de89d2c9a4f